yak/rest_social_network/views.py

```diff
@@ -13,6 +14,7 @@
 
 
 User = get_user_model()
+drf_version = [int(num) for num in rest_framework.__version__.split('.')]
```

Comment:
Could `[int(num) for num in rest_framework.__version__.split('.')]` be put into a utility that just gets passed the module whose version we're looking for?